build.sbt

```diff
@@ -21,11 +21,18 @@ ThisBuild / tlJdkRelease := Some(8)
 
 ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("snapshots")
 
-lazy val precompile = taskKey[Unit]("runs Calico-specific pre-compile tasks")
+/*lazy val precompile = taskKey[Unit]("runs Calico-specific pre-compile tasks")*/
 
-precompile := DomDefsGenerator.cachedGenerate()
+/*precompile := DomDefsGenerator.cachedGenerate((calico / sourceManaged).value)*/
 
-(Compile / compile) := ((Compile / compile) dependsOn precompile).value
+/*(Compile / compile) := ((Compile / compile) dependsOn precompile).value*/
+
+Compile / sourceGenerators += Def.task {
+  val calicoSourceManaged = (calico / sourceManaged).value
+  DomDefsGenerator.cachedGenerate(calicoSourceManaged)
+  val finder: PathFinder = calicoSourceManaged ** "*.scala"
+  finder.get
+}.taskValue
```

Comment:
This should be defined in the `.settings()` for the calico subproject, right?

Reply:
Ahhh, yes, you are perfectly right! Seems to work now. I need some sleep, see you tomorrow :)